android/src/paper/java/com/swmansion/rnscreens/FabricEnabledViewGroup.kt

```diff
@@ -2,8 +2,12 @@ package com.swmansion.rnscreens
 
 import android.view.ViewGroup
 import com.facebook.react.bridge.ReactContext
+import com.facebook.react.uimanager.FabricViewStateManager
 
 abstract class FabricEnabledViewGroup constructor(context: ReactContext?) : ViewGroup(context) {
+
+    val fabricViewStateManager get() = null as FabricViewStateManager
```

Comment:
Also one more thing actually 😅 as `fabricViewStateManager` could be nullable, I'll go for typing this val as `FabricViewStateManager?`. I know that we're protecting our backs with `BuildConfig.IS_NEW_ARCHITECTURE_ENABLED`, but I'm afraid this condition may fail at some point and will crash the app. Also, it's more logical for me to make this `optional`, as this val is still reachable from main code and can be used in paper implementation (which is not correct, but it won't crash the app if we use that unintentionally). What do you think?